execution_workflows/Roar/.Rhistory

```diff
@@ -0,0 +1,2 @@
+library(biomaRt)
```

Comment:
Like `.RData` this should not go inside the repo. In fact these may be good candidates to add to `.gitignore`